### PR TITLE
To scope

### PR DESCRIPTION
**Pull request by ckjones818**
As I mentioned in slack, I didn't add default_scope functionality because everything I've seen online has suggested that it's bad practice/it didn't really succeed in making code any DRYer. Also I hope that my implementation matches up with what people were thinking in theory -- I can go back and make any needed changes obvs

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,7 +1,17 @@
+# module for user access
+module Current
+  thread_mattr_accessor :user
+end
+
 # Application Controller
 class ApplicationController < ActionController::Base
   before_action :authenticate_user!
-  def after_sign_in_path_for(_resource)
```
Comment (lilyosah) [inline app/controllers/application_controller.rb:4]: @ckjones818 This doesn't break redirection after sign in right?

**Review by lilyosah — CHANGES_REQUESTED** [review]
If you could change that one name for clarity everything else looks great! Good work

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ckjones818) [reply app/controllers/application_controller.rb:4]: Nope

File: app/models/task.rb [hunk 2]
```diff
@@ -7,6 +7,15 @@ class Task < ApplicationRecord
...
   validate :start_not_after_due?
 
+  scope :get_user, -> { where('user_id = ?', Current.user.id) }
+  scope :find_task, -> { find(params[:id]) }
+  scope :today, ->(position, equality) { where("#{position} #{equality} ?", DateTime.now.to_date.to_formatted_s(:db)) }
+  scope :tomorrow, lambda { |position, equality|
+                     where("#{position} #{equality} ?", DateTime.now.to_date.tomorrow.to_formatted_s(:db))
+                   }
+  scope :from_params, ->(position, equality, to_datetime) { where("#{position} #{equality} ?", to_datetime) }
+  scope :order_by, ->(direction) { order("due #{direction}") }
```
[the remark this answers is not in the document]
Comment (ckjones818) [reply app/models/task.rb:17]: Sure will do

**Review by lilyosah — APPROVED** [review]
Awesome, thanks Colin! ✨

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (lilyosah) [reply app/controllers/application_controller.rb:4]: @ckjones818

**Pull request merged**